scripts/README.md

```
@@ -118,6 +148,17 @@ Localizados em `build/` - documentados em [build/README.md](../build/README.md)
 
 ## 📊 Resumo
 
-- **Total de scripts:** 4 PowerShell essenciais
+- **Total de scripts:** 5 PowerShell + 1 SQL
 - **Foco:** Migrations, seed de dados, export de API
 - **Filosofia:** Apenas scripts com utilidade clara e automação
+
+### Estratégia de Seeding
+| Tipo | Quando | Propósito | Exemplo |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)